lib/game_teams_manager.rb

```diff
@@ -26,39 +27,12 @@ def total_game_teams(filtered_game_teams = @game_teams)
     filtered_game_teams.count
   end
 
-  def away_games_by_team
-    away_games.group_by do |game_team|
-      game_team.team_id
-    end
-  end
-
-  # USE THIS OR INDIVIDUAL ONES?
-  def home_or_away_games(where = "home")
+  def home_or_away_games(hoa)
     @game_teams.select do |game|
-      game.hoa == where
-    end
-    require "pry"; binding.pry
-  end
-
-  def away_games
-    @game_teams.select do |game_team|
-      game_team.hoa == "away"
-    end
-  end
-
-  def home_games_by_team
-    home_games.group_by do |game_team|
-      game_team.team_id
+      game.hoa == hoa
     end
   end
 
-  def highest_scoring_visitor
```

Comment:
Jake, have you confirmed this is passing spec harness?

Reply:
Yes, all 4 of highest/lowest visitor/home pass spec harness